src/views/portal/AcceptedMessage.tsx

```diff
@@ -0,0 +1,139 @@
+import styles from "views/portal/Portal.module.scss";
+
+function AcceptedMessage() {
+	return (
+		<>
+			<h4>Waiver</h4>
+			<p>
+				ALL ATTENDEES, please review our{" "}
+				<a href="https://docs.google.com/document/u/1/d/e/2PACX-1vRPuYBKQ2YR64FVzDKunDeVkeFGD573hSqfTmf8JwvyhlvPQPEnovxhlYbal45WI3k3Jr2iLNRzz1Hp/pub">
+					Code of Conduct
+				</a>{" "}
+				and sign the Event Waiver prior to the event. You will receive the
+				waiver via email closer to the event date.{" "}
+				<strong className={styles.red_text}>
+					<i>
+						<u>
+							You must complete the waiver in order to participate in Hack at
+							UCI.
+						</u>
+					</i>
+				</strong>
+			</p>
+
+			<h4>Communication</h4>
+			<p>
+				Communication during the event will primarily be done through Slack. You
+				will receive an invite via email closer to the event date.
+			</p>
+
+			<h4>Team Formation</h4>
+			<p>
+				Still looking for teammates? Attend our two team formation events,
+				Thursday (2/2) 7-8 PM in DBH 6011 and Friday (2/3) 7-8 PM in Student
+				Center Pacific Ballroom. You can also create an introductory slide{" "}
+				<a href="https://docs.google.com/presentation/d/1qjAzK9ocrI5Lg2D8l4VnE38CWDU2bW2v3XqAROA-vR4/edit#slide=id.g1ecf0247681_1_277">
+					here
+				</a>{" "}
```

Comment:
Suggestion: instead of writing "here", we could use more descriptive link text, e.g.
> create an introductory slide [on our team formation slides](https://github.com/HackAtUCI/HackUCI-Site/pull/78/files#)